### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by filipblnt**
Draft PR for finishing the README.

### CODE REVIEW

**Comment by noelmace** [comment]
I'm using this PR as an example for https://github.com/blindnet-io/openness-framework/pull/33.

As I'm now a co-author, it LGTM, obviously :smile: So I'm making @m4rk055 the reviewer and assignee.

Please keep this PR review about its specificities only, and give any feedback about the template itself in https://github.com/blindnet-io/openness-framework/pull/33.

**Review by noelmace — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,94 @@
-# The javascript web client SDK for blindnet
+<h1 align="center">
+  blindnet devkit<br />Javascript Client SDK
+</h1>
+
+<p align=center><img src="https://user-images.githubusercontent.com/7578400/163277439-edd00509-1d1b-4565-a0d3-49057ebeb92a.png#gh-light-mode-only" height="80" /></p>
+<p align=center><img src="https://user-images.githubusercontent.com/7578400/163549893-117bbd70-b81a-47fd-8e1f-844911e48d68.png#gh-dark-mode-only" height="80" /></p>
+
+<p align="center">
+  <strong>JavaScript <a href="https://blindnet.dev/docs/extra/glossary/#client-side-sdk">client-side SDK</a> for e2e encryption and users management.</strong>
```
Comment (m4rk055) [inline README.md:9]: Structure of the docs changed a bit. Remove `/extra` from every _blindnet.dev_ link in the README

File: README.md [hunk 2]
```diff
@@ -1 +1,94 @@
...
+## About
+
+This is the JavaScript client-side SDK component of [blindnet devkit][devkit] allowing you to:
+
+- [encrypt and decrypt data](https://blindnet.dev/docs/extra/glossary/#encrypted-data) for users,
+- and manage [user](https://blindnet.dev/docs/extra/glossary/#key-pair) and [data](https://docs.blindnet.io/docs/extra/glossary/#encrypted-data-key) encryption keys.
+
+## Get Started
+
+:rocket: Check out our [Quick Start Guide](http://localhost:3000/docs/quickstart) to get started in a snap.
```
Comment (m4rk055) [inline README.md:37]:
```suggestion
:rocket: Check out our [Quick Start Guide](https://blindnet.dev/docs/quickstart) to get started in a snap.
```

File: README.md [hunk 3]
```diff
@@ -1 +1,94 @@
...
+
+Stay up to date with new releases and projects, learn more about how to protect your privacy and that of our users, and share projects and feedback with our team.
+
+- [Join our Slack Workspace][chat] to chat with the blindnet community and team
+- Follow us on [Twitter][twitter] to stay up to date with the latest news
+- Check out our [Openness Framework][openness] and [Product Management][product] on Github to see how we operate and give us feedback.
+
+## License
+
+The blindnet devkit sdk-javascript-client is available under [MIT](license) (and [here](https://github.com/blindnet-io/openness-framework/blob/main/docs/decision-records/DR-0001-oss-license.md) is why).
```
Comment (m4rk055) [inline README.md:71]:
```suggestion
The blindnet devkit sdk-javascript-client is available under [MIT][license] (and [here](https://github.com/blindnet-io/openness-framework/blob/main/docs/decision-records/DR-0001-oss-license.md) is why).
```

**Review by m4rk055 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (noelmace) [reply README.md:37]: oupsy :sweat_smile:

**Pull request merged**